cogs/moderation/moderation.py

```diff
@@ -184,6 +183,51 @@ async def tempban(self, ctx: Context, member: discord.Member, length : CustomTim
 
             await ctx.send(embed=embed)
 
+    @commands.command(aliases=['purge', 'del', 'delete'])
+    @commands.guild_only()
+    @commands.has_permissions(manage_messages=True)
+    async def clear(self, ctx: Context, channel: Optional[discord.TextChannel], amount: int):
+        channel = channel or ctx.channel
+
+        await ctx.send(f"now purging {amount} messages...")
+
+        await channel.purge(limit=amount + 1)
+        self.bot.dispatch("purge", author=ctx.author)
+
+    @commands.command(name="unban")
+    @commands.guild_only()
+    @commands.has_permissions(ban_members=True)
+    async def _unban(self, ctx, id: int, *, reason="No Reason Provided"):
+        try:
+            user = discord.Object(id=id)
+            cid = next(self.bot.idgen)
```

Comment:
Should we be treating an unban as a case and inserting data for it into the table? I feel like we should have some column in the table like "pardoned" or something, and an unban merely sets that field to True instead of creating a completely new case in the table. Maybe we should discuss this again?